jupyter-book/introduction/raw_data_processing.md

```diff
@@ -1,52 +1,397 @@
+
+In this example, we will use _chromosome 5_ of the human genome and its related gene annotations as the reference, which is a subset of the [Human reference, GRCh38 (GENCODE v32/Ensembl 98) reference](https://support.10xgenomics.com/single-cell-gene-expression/software/release-notes/build#GRCh38_2020A) from the 10x Genomics. Correspondingly, we extract the subset of reads that map to the generated reference from a [human brain tumor dataset](https://www.10xgenomics.com/resources/datasets/200-sorted-cells-from-human-glioblastoma-multiforme-3-lt-v-3-1-3-1-low-6-0-0) from the 10x Genomics. 
+
+[`Alevin-fry`](https://alevin-fry.readthedocs.io/en/latest/){cite}`He2022` is a fast, accurate, and memory-frugal single-cell data processing tool. Here we show how to process the example dataset described above using `alevin-fry` from the command line. While simplified [wrapper scripts](https://github.com/COMBINE-lab/usefulaf) and [workflows](https://github.com/COMBINE-lab/quantaf) exist to process data with this tool, we will describe here the full set of commands to outline 
+where the steps described in this section occur and to convey what different processing options are possible. These commands will be run from the command line, and [`conda`](https://docs.conda.io/en/latest/) will be used for installing all of the software required for running this example.
+
+(raw-proc:example-prep)=
+### Preparation
+
+Before we start, we create a conda environment in the terminal and install the required packages. In this procedure, we will install [`alevin-fry`](https://alevin-fry.readthedocs.io/en/latest/), [`salmon`](https://salmon.readthedocs.io/en/latest/) and [`pyroe`](https://github.com/COMBINE-lab/pyroe).
+
+```bash
+conda create -n af_xmpl -y -c bioconda python=3.9 salmon alevin-fry pyroe
+conda activate af_xmpl
+```
+
+```{admonition} Note on using an M1/M2-based device
+
+Conda does not currently build most packages natively for Apple silicon. Therefore, if you 
+are using a non-Intel-based Apple computer (e.g., with an M1(Pro/Max/Ultra) or M2 chip), you 
+should make sure to specify that your environment uses the Rosetta2 translation layer.  
+To do this, you can replace the above commands with the following (instructions adopted 
+from [here](https://github.com/Haydnspass/miniforge#rosetta-on-mac-with-apple-silicon-hardware)):
+
+```bash
+CONDA_SUBDIR=osx-64 conda create -n af_xmpl -y -c bioconda python=3.9 salmon alevin-fry pyroe   # create a new environment
+conda activate af_xmpl
+conda env config vars set CONDA_SUBDIR=osx-64  # subsequent commands use intel packages
+```
+
+Next, we create a working directory `af_xmpl_run`, and download and uncompress the example dataset from a remote host.
+
+```bash
+# create a working dir 
+mkdir af_xmpl_run && cd af_xmpl_run
+
+# download the example dataset
```

Comment:
I can read this, biologists who just learned their first bash commands can not. I'd suggest that you expand the comment with more details on the piping (2. step aka the untar).

Reply:
We have added more details on this.